example/src/main/kotlin/com.expedia.graphql.sample/directives/DirectiveWiring.kt

```diff
@@ -0,0 +1,17 @@
+package com.expedia.graphql.sample.directives
+
+import com.google.common.base.CaseFormat
+import graphql.schema.idl.SchemaDirectiveWiring
+import graphql.schema.idl.SchemaDirectiveWiringEnvironment
+import kotlin.reflect.KClass
+
+interface DirectiveWiring : SchemaDirectiveWiring {
+
+    companion object {
+        fun getDirectiveName(kClass: KClass<out Annotation>): String =
```

Comment:
there is no need for companion object, lets move up this method outside of interface (lets keep it in the same file though)